ipyleaflet/leaflet.py

```diff
@@ -9,13 +11,15 @@
 
 from traitlets import (
     Float, Unicode, Int, Tuple, List, Instance, Bool, Dict, Enum,
-    link, observe, default, validate, TraitError, Union
+    link, observe, default, validate, TraitError, Union, Any
```

Comment:
You can remove this change, I suppose you needed `Any` at some point but you don't need it anymore.